src/reference/interactive/index.pug

```diff
@@ -318,26 +334,26 @@ block reference
+    p A reply is sent from a recipient back to the caller to informing about the result after executing the method that was sent. It can contain a #[code result] or an #[code error] which indicates what went wrong.
 
     p For full packet implementation details please refer to the protocol specification which you can download #[a(href='/reference/interactive/protocol/protocol.pdf' target='_blank') here].
 
     h3#compression Compression
-    p By default packets on the wire are transmitted as plaintext, but the Game Client can opt to use #[a(href='https://tools.ietf.org/html/rfc1952' target='_blank') GZIP] or #[a(href='https://lz4.github.io/lz4/' target='_blank') LZ4] compression. To do this, the Game Client must call a method providing its supported compression formats. The server will then respond with its chosen compression format.
+    p By default, packets on the wire are transmitted as plain text, but the Game Client can opt to use #[a(href='https://tools.ietf.org/html/rfc1952' target='_blank') GZIP] or #[a(href='https://lz4.github.io/lz4/' target='_blank') LZ4] compression. To do this, the Game Client must call a method providing its supported compression formats. The server will then respond with its chosen compression format.
 
     h3#authentication Authentication
-    p A Game Client needs to authenticate as a Beam user when establishing an interactive session. Two authentication methods are available.
+    p A Game Client needs to authenticate as a Beam user when establishing an interactive session. There are two authentication methods available.
 
     h4(id='oauth-2.0') OAuth 2.0
     p Beam supports #[a(href='https://tools.ietf.org/html/rfc6749' target='_blank') OAuth 2.0] flows which enable you to get a valid #[a(href='https://tools.ietf.org/html/rfc6750' target='_blank') OAuth 2.0 Bearer] token. Tokens can be passed in the #[code Authorization] header when you initiate a connection to the interactive service.
 
-    p The only required scope for an interactive connection is #[code interactive:robot:self]. For more information about Beam's OAuth visit the #[a(href='/reference/oauth/index.html' target='_blank') OAuth reference page].
+    p The only required scope for an interactive connection is #[code interactive:robot:self]. For more info about Beam's OAuth, go to #[a(href='/reference/oauth/index.html' target='_blank') OAuth reference page].
 
     h4#xtoken XToken
-    p You can provide an XBL XToken in the #[code Authorization] header when you initiate a connection to the interactive service. This authentication method is leveraged for Universal Windows Platform applications that are Xbox Live enabled, as well as games on Xbox One
+    p You can provide a Xbox Live's XToken in the #[code Authorization] header when you initiate a connection to the interactive service. This authentication method is useful for Universal Windows Platform (UWP) applications that are Xbox Live enabled, as well as games on Xbox One.
```

Comment:
Probably "You can provide a Xbox Live XToken" The tokens aren't technically owned by Xbox Live afaik